packages/create-react-app/createReactApp.js

```diff
@@ -143,7 +143,7 @@ if (typeof projectName === 'undefined') {
   process.exit(1);
 }
 
-function printValidationResults(results) {
+printValidationResults = (results) => {
```

Comment:
If you change this function declaration to an arrow function, you should replace the `function` keyword with the declaration of a variable, like so: `const printValidationResults = (results) => { ... }`.